developmentalparsers/w2dynamics/parser.py

```diff
@@ -38,20 +38,20 @@ def init_quantities(self):
         section_quantities = [
             Quantity(
                 'parameter', f'{re_n} *(\w+ *= *.+)',
-                repeats=True, str_operation=lambda x: x.split('=', 1))
```

Comment:
we can also do `lambda x: [v.strip() for v in x.split('=', 1)]`

Reply:
What is the advantage?